src/blocks/opentable/save.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * WordPress dependencies.
+ */
+const {__} = wp.i18n;
+
+const OpenTableSave = function( props ) {
+	const {
+		attributes: {
+			restaurantId,
+			dateLabel,
+			timeLabel,
+			partySizeLabel,
+			submitButtonText
+		},
+	} = props;
+
+	return (
+
+		<div className="nova-opentable">
+			<form method="get" className="nova-opentable__form" action="//www.opentable.com/restaurant-search.aspx" target="_blank">
+				<div className="nova-opentable__wrapper">
+					<div className="nova-opentable__date nova-opentable__input-wrap">
+						<label>{dateLabel}</label>
+						<input id="datepicker" name="startDate" className="otb-date otb-input" type="text" value="8/30/2019" readOnly="" autoComplete="off"/>
```

Comment:
Use the date today for the initial value of the field.

Reply:
Done.